src/zope/testing/test_renormalizing.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from zope.testing.renormalizing import strip_dottedname_from_traceback
+
+
+class Exception2To3(unittest.TestCase):
+
+    def test_strip_dottedname(self):
+        string = """\
+Traceback (most recent call last):
+foo.bar.FooBarError: requires at least one argument."""
+        expected = """\
+Traceback (most recent call last):
+FooBarError: requires at least one argument."""
```

Comment:
Please use `textwrap.dedent()` here and in all the other tests in this file, to make this readable.